sdk/keyvault/Azure.Security.KeyVault.Administration/src/PreRestoreOperationParameters.cs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Security.KeyVault.Administration.Models
+{
+    internal partial class PreRestoreOperationParameters
```

Comment:
Why did you mark this internal? It's probably not compiling since the method that declares a parameter of this type is necessarily public.

Reply:
@heaths I don't think we need to extend the generated model. I am curious, is there a reason for adding this empty partial class in the custom layer? I noticed that [RestoreOperationParameters](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Administration/src/RestoreOperationParameters.cs) also has an empty partial class in the `src/` directory.

Reply:
I don't think @christothes wanted to expose them directly. He implemented almost all of the Administration package, so he could answer best.

Reply:
I don't recall now, but from the looks of it, that partial class for `RestoreOperationParameters` was probably an autorest model transform at some point that was never removed. It doesn't appear to do anything.